packages/driver/src/cypress.js

```diff
@@ -596,6 +598,19 @@ class $Cypress {
     return throwPrivateCommandInterface('addUtilityCommand')
   }
 
+  get currentTest () {
+    const r = this.cy.state('runnable')
+
+    if (!r) {
+      const invocationStack = $stackUtils.getInvocationDetails(this.cy.state('specWindow'), this.config)?.stack
+
+      throw errByPath(errs.currentTest.outside_test)
+      .setUserInvocationStack(invocationStack)
+    }
+
+    return r && r.ctx.currentTest || r
```

Comment:
hm, if `r` is falsy, this getter function will have already thrown on line 607, so isn't this exactly the same as...

```suggestion
    return r.ctx.currentTest
```

?

Reply:
o, `r.ctx.currentTest || r`, right